2016-08/august-24.md

```diff
@@ -0,0 +1,43 @@
+## August 24 ([discuss](https://github.com/webpack/meeting-notes/pull/11))
+
+### Attendees
+
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Tobias Koppers](http://github.com/sokra)
+
+
+#### Attendee Notes:
+Johannes is in Iceland at JSConf Iceland (Good luck!), and Juho was unable to make the meeting today. 
+
+### Agenda
+
+#### Documentation Updates
+
+* We were able to reach [Marco Pai](https://github.com/MarcoPai), however the offer he kindly extended to us for puchase of  webpack.io was too steep. We may consider keeping the existing domain, as the traffic is extremely high. We decided to talk about this after documentation has been a little bit more fleshed out. 
+
+* We have had some more pull requests for documentation, however on the current pace we are at, it is most unlikely that we will not be able to release the beta tag from webpack 2 just yet because we have not fullfilled our [milestone/plan of action](https://github.com/webpack/webpack.io/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Webpack+2+-+Documentation+MVP%22). We are working on getting the site deployed, so that it is easier for people to see and use the new documentation page. Contributions should be as easy as getting a github link for the document you are looking at and then commiting the changes, and submitting a PR. 
+
+#### Reported Issues
+
+**Tree Shaking / Optimization**  
+* We discussed briefly our disappointment that [UglifyJS2](https://github.com/mishoo/UglifyJS2/issues/1261) is not willing to work on a solution to to remove side-affects from transpiled unused ES6 exported classes transpiled to ES5 (by both Typescript and Babel). [#2867](https://github.com/webpack/webpack/issues/2867). 
```

Comment:
Now that [closure-compiler-js](https://github.com/google/closure-compiler-js) is a thing, I wonder if it'd be worth investigating using that. This may be a ridiculous suggestion because I don't know much about closure compiler, but if I understand correctly they have impressive dead code elimination capabilities.